godtools/Managers/LanguagesManager.swift

```diff
@@ -43,7 +43,7 @@ class LanguagesManager: NSObject {
     func loadFromRemote() -> Promise<[Language]> {
         showNetworkingIndicator()
         
-        return Alamofire.request(URL(string: "\(GodToolsConstants.kApiBase)/\(path)")!).responseData().then { data -> Promise<[Language]> in
+        return Alamofire.request(URL(string: "\(GTConstants().kApiBase)/\(path)")!).responseData().then { data -> Promise<[Language]> in
```

Comment:
If you use `static` in the struct you are going to be able to access like `GTConstants. kApiBase`